allensdk/brain_observatory/behavior/data_objects/metadata/ophys_experiment_metadata/emission_lambda.py

```diff
@@ -0,0 +1,21 @@
+from pynwb import NWBFile
+
+from allensdk.brain_observatory.behavior.data_objects import DataObject
+from allensdk.brain_observatory.behavior.data_objects.base\
+    .readable_interfaces.nwb_readable_interface import \
+    NwbReadableInterface
+
+
+class EmissionLambda(DataObject, NwbReadableInterface):
```

Comment:
how can we have something that is readable but not writable?
is this a TODO, or is emission lambda written somewhere else?

Reply:
probably you will tell me it is part of `metadata`
for such objects, perhaps we can allude to that in a docstring

Reply:
The metadata gets split up when it gets written to NWB. `emission_lambda` for example is written [here](https://github.com/AllenInstitute/AllenSDK/pull/2153/files#diff-f90c0c6a9702634078317391ad0fed18115c66e9481014bf6d906de7bf9ccc35R144).

There might be a better to align this with how NWB stores the data.